README.md

```diff
@@ -0,0 +1,68 @@
+# hsk
+
+The `hsklogin` script uses the macOS ssh-agent with the yubikey, 
+so that you don't have to type in the PIN for every git pull.
+
+## Prerequirements
+
+Set up your yubikey for github SSH and test that it works correctly 
+(follow instructions at https://w3.ibm.com/#/support/article/hsk/github).
+
+
+## Installing
+
+```
+git clone https://github.com/mrszymon/hsk
+cd hsk
+sudo cp hsklogin /usr/local/bin
```

Comment:
Why make a copy instead of linking?

Reply:
Pretty normal approach, but I will defer to you and @mrszymon

Reply:
From experience, the directory that gets cloned to will be deleted or moved or something at some point and suddenly the link stops working :-)

Reply:
From experience, when things get copied, people keep using outdated versions without ever noticing ;-)
Maybe the documentation could suggest both ways ;-)